tests/contrib/flask/test_errorhandler.py

```diff
@@ -15,6 +15,8 @@ def test_default_404_handler(self):
         # Make our 404 request
```

Comment:
would probably be good to introduce a snapshot test case before this PR to see the real impact on the traces produced

Reply:
Done in #3971